### ISSUE TITLE
gcloud commands returns an error

### ISSUE THREAD

**Issue by UriKatsirPrivate** [google-github-actions/auth#220]
### TL;DR

When using the auth action with a gcloud command, the gcloud command fails.

### Expected behavior

gcloud command running successfully 

### Observed behavior

Run gcloud builds submit /floor/ --tag europe-west4-docker.pkg.dev/landing-zone-demo-341118/cloud-run-source-deploy/feature-environment:latest
  gcloud builds submit /floor/ --tag europe-west4-docker.pkg.dev/landing-zone-demo-341118/cloud-run-source-deploy/feature-environment:latest
  
  
  shell: /usr/bin/bash -e {0}
  env:
    PROJECT_ID: landing-zone-demo-341118
    SERVICE: floor
    REGION: europe-west4
    CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-67a4cbb79684a778.json
    GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-67a4cbb79684a778.json
    GOOGLE_GHA_CREDS_PATH: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-67a4cbb79684a778.json
    CLOUDSDK_CORE_PROJECT: landing-zone-demo-341118
    CLOUDSDK_PROJECT: landing-zone-demo-341118
    GCLOUD_PROJECT: landing-zone-demo-341118
    GCP_PROJECT: landing-zone-demo-341118
    GOOGLE_CLOUD_PROJECT: landing-zone-demo-341118
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
ERROR: (gcloud.builds.submit) There was a problem refreshing your current auth tokens: ('Unable to acquire impersonated credentials: No access token or invalid expiration in response.', '{\n  "error": {\n    "code": 403,\n    "message": "The caller does not have permission",\n    "status": "PERMISSION_DENIED"\n  }\n}\n')
Please run:

  $ gcloud auth login

to obtain new credentials.

If you have already logged in with a different account:

    $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
Error: Process completed with exit code 1.

### Action YAML

```yaml
on:
  push:
      branches: [ main ]
  pull_request:
      branches: [ main ]
name: Build and Deploy to Cloud Run
env:
  PROJECT_ID: "MY-PROJECT-ID"
  SERVICE: "floor"
  REGION: "europe-west4"

jobs:
  deploy:
    runs-on: ubuntu-latest

    # Add "id-token" with the intended permissions.
    permissions:
      contents: 'read'
      id-token: 'write'
    steps:
    - name: Checkout
      uses: actions/checkout@v3

    # Configure Workload Identity Federation and generate an access token.
    - id: 'auth'
      name: 'Authenticate to Google Cloud'
      uses: 'google-github-actions/auth@v0'
      with:
        workload_identity_provider: 'projects/123654879/locations/global/workloadIdentityPools/github-pool/providers/my-provider'
        service_account: 'workflows-sa@MY-PROJECT-ID.iam.gserviceaccount.com'
        create_credentials_file: true

    # Setup gcloud CLI
    - name: Set up Cloud SDK
      uses: google-github-actions/setup-gcloud@v0

    - id: 'gcloud'
      name: Build and Push Container
      run: |-
        gcloud builds submit /floor/ --tag europe-west4-docker.pkg.dev/landing-zone-demo-341118/cloud-run-source-deploy/feature-environment:latest
```

### Log output

```text
2022-09-01T18:21:38.5170474Z Requested labels: ubuntu-latest
2022-09-01T18:21:38.5170511Z Job defined at: UriKatsirPrivate/Feature-Environment/.github/workflows/deploy.yaml@refs/heads/main
2022-09-01T18:21:38.5170530Z Waiting for a runner to pick up this job...
2022-09-01T18:21:39.3312920Z Job is waiting for a hosted runner to come online.
2022-09-01T18:21:42.8310605Z Job is about to start running on the hosted runner: Hosted Agent (hosted)
2022-09-01T18:21:46.9350434Z Current runner version: '2.296.1'
2022-09-01T18:21:46.9375585Z ##[group]Operating System
2022-09-01T18:21:46.9376130Z Ubuntu
2022-09-01T18:21:46.9376475Z 20.04.4
2022-09-01T18:21:46.9376688Z LTS
2022-09-01T18:21:46.9376962Z ##[endgroup]
2022-09-01T18:21:46.9377263Z ##[group]Runner Image
2022-09-01T18:21:46.9377568Z Image: ubuntu-20.04
2022-09-01T18:21:46.9377905Z Version: 20220828.1
2022-09-01T18:21:46.9378395Z Included Software: https://github.com/actions/runner-images/blob/ubuntu20/20220828.1/images/linux/Ubuntu2004-Readme.md
2022-09-01T18:21:46.9378948Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu20%2F20220828.1
2022-09-01T18:21:46.9379371Z ##[endgroup]
2022-09-01T18:21:46.9379686Z ##[group]Runner Image Provisioner
2022-09-01T18:21:46.9379998Z 1.0.0.0-main-20220825-1
2022-09-01T18:21:46.9380288Z ##[endgroup]
2022-09-01T18:21:46.9380902Z ##[group]GITHUB_TOKEN Permissions
2022-09-01T18:21:46.9381377Z Contents: read
2022-09-01T18:21:46.9381824Z Metadata: read
2022-09-01T18:21:46.9382134Z ##[endgroup]
2022-09-01T18:21:46.9386012Z Secret source: Actions
2022-09-01T18:21:46.9386465Z Prepare workflow directory
2022-09-01T18:21:47.0371336Z Prepare all required actions
2022-09-01T18:21:47.0556337Z Getting action download info
2022-09-01T18:21:47.2799555Z Download action repository 'actions/checkout@v3' (SHA:2541b1294d2704b0964813337f33b291d3f8596b)
2022-09-01T18:21:47.5876014Z Download action repository 'google-github-actions/auth@v0' (SHA:dac4e13deb3640f22e3ffe758fd3f95e6e89f712)
2022-09-01T18:21:47.7350686Z Download action repository 'google-github-actions/setup-gcloud@v0' (SHA:877d4953d2c70a0ba7ef3290ae968eb24af233bb)
2022-09-01T18:21:48.0958292Z ##[group]Run actions/checkout@v3
2022-09-01T18:21:48.0958693Z with:
2022-09-01T18:21:48.0959116Z   repository: UriKatsirPrivate/Feature-Environment
2022-09-01T18:21:48.0959882Z   token: ***
2022-09-01T18:21:48.0960172Z   ssh-strict: true
2022-09-01T18:21:48.0960565Z   persist-credentials: true
2022-09-01T18:21:48.0960916Z   clean: true
2022-09-01T18:21:48.0961205Z   fetch-depth: 1
2022-09-01T18:21:48.0961569Z   lfs: false
2022-09-01T18:21:48.0961896Z   submodules: false
2022-09-01T18:21:48.0962234Z   set-safe-directory: true
2022-09-01T18:21:48.0962588Z env:
2022-09-01T18:21:48.0962950Z   PROJECT_ID: landing-zone-demo-341118
2022-09-01T18:21:48.0963320Z   SERVICE: floor
2022-09-01T18:21:48.0963667Z   REGION: europe-west4
2022-09-01T18:21:48.0964006Z ##[endgroup]
2022-09-01T18:21:48.3776800Z Syncing repository: UriKatsirPrivate/Feature-Environment
2022-09-01T18:21:48.3779590Z ##[group]Getting Git version info
2022-09-01T18:21:48.3780955Z Working directory is '/home/runner/work/Feature-Environment/Feature-Environment'
2022-09-01T18:21:48.3799794Z [command]/usr/bin/git version
2022-09-01T18:21:48.3917978Z git version 2.37.2
2022-09-01T18:21:48.3942732Z ##[endgroup]
2022-09-01T18:21:48.3960781Z Temporarily overriding HOME='/home/runner/work/_temp/da9f42d8-ddcd-43ba-bc0a-44d6c7659ce9' before making global git config changes
2022-09-01T18:21:48.3961998Z Adding repository directory to the temporary git global config as a safe directory
2022-09-01T18:21:48.3965803Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/Feature-Environment/Feature-Environment
2022-09-01T18:21:48.4009264Z Deleting the contents of '/home/runner/work/Feature-Environment/Feature-Environment'
2022-09-01T18:21:48.4013444Z ##[group]Initializing the repository
2022-09-01T18:21:48.4017013Z [command]/usr/bin/git init /home/runner/work/Feature-Environment/Feature-Environment
2022-09-01T18:21:48.4091054Z hint: Using 'master' as the name for the initial branch. This default branch name
2022-09-01T18:21:48.4095886Z hint: is subject to change. To configure the initial branch name to use in all
2022-09-01T18:21:48.4099121Z hint: of your new repositories, which will suppress this warning, call:
2022-09-01T18:21:48.4101946Z hint: 
2022-09-01T18:21:48.4105801Z hint: 	git config --global init.defaultBranch <name>
2022-09-01T18:21:48.4108553Z hint: 
2022-09-01T18:21:48.4111678Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2022-09-01T18:21:48.4114746Z hint: 'development'. The just-created branch can be renamed via this command:
2022-09-01T18:21:48.4118399Z hint: 
2022-09-01T18:21:48.4121369Z hint: 	git branch -m <name>
2022-09-01T18:21:48.4137623Z Initialized empty Git repository in /home/runner/work/Feature-Environment/Feature-Environment/.git/
2022-09-01T18:21:48.4158717Z [command]/usr/bin/git remote add origin https://github.com/UriKatsirPrivate/Feature-Environment
2022-09-01T18:21:48.4227720Z ##[endgroup]
2022-09-01T18:21:48.4228658Z ##[group]Disabling automatic garbage collection
2022-09-01T18:21:48.4232222Z [command]/usr/bin/git config --local gc.auto 0
2022-09-01T18:21:48.4262318Z ##[endgroup]
2022-09-01T18:21:48.4263348Z ##[group]Setting up auth
2022-09-01T18:21:48.4270215Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-09-01T18:21:48.4303567Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-09-01T18:21:48.4624897Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-09-01T18:21:48.4659879Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-09-01T18:21:48.4884922Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2022-09-01T18:21:48.4931672Z ##[endgroup]
2022-09-01T18:21:48.4932708Z ##[group]Fetching the repository
2022-09-01T18:21:48.4934162Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +1e34e46b5a72649fb69427f76647278dff8e5e99:refs/remotes/origin/main
2022-09-01T18:21:48.7519389Z remote: Enumerating objects: 23, done.        
2022-09-01T18:21:48.7523808Z remote: Counting objects:   4% (1/23)        
2022-09-01T18:21:48.7524760Z remote: Counting objects:   8% (2/23)        
2022-09-01T18:21:48.7525248Z remote: Counting objects:  13% (3/23)        
2022-09-01T18:21:48.7526051Z remote: Counting objects:  17% (4/23)        
2022-09-01T18:21:48.7526500Z remote: Counting objects:  21% (5/23)        
2022-09-01T18:21:48.7527281Z remote: Counting objects:  26% (6/23)        
2022-09-01T18:21:48.7527735Z remote: Counting objects:  30% (7/23)        
2022-09-01T18:21:48.7528512Z remote: Counting objects:  34% (8/23)        
2022-09-01T18:21:48.7528967Z remote: Counting objects:  39% (9/23)        
2022-09-01T18:21:48.7529740Z remote: Counting objects:  43% (10/23)        
2022-09-01T18:21:48.7530195Z remote: Counting objects:  47% (11/23)        
2022-09-01T18:21:48.7530977Z remote: Counting objects:  52% (12/23)        
2022-09-01T18:21:48.7531425Z remote: Counting objects:  56% (13/23)        
2022-09-01T18:21:48.7532193Z remote: Counting objects:  60% (14/23)        
2022-09-01T18:21:48.7532668Z remote: Counting objects:  65% (15/23)        
2022-09-01T18:21:48.7533514Z remote: Counting objects:  69% (16/23)        
2022-09-01T18:21:48.7534700Z remote: Counting objects:  73% (17/23)        
2022-09-01T18:21:48.7535210Z remote: Counting objects:  78% (18/23)        
2022-09-01T18:21:48.7535921Z remote: Counting objects:  82% (19/23)        
2022-09-01T18:21:48.7536403Z remote: Counting objects:  86% (20/23)        
2022-09-01T18:21:48.7537263Z remote: Counting objects:  91% (21/23)        
2022-09-01T18:21:48.7537853Z remote: Counting objects:  95% (22/23)        
2022-09-01T18:21:48.7538427Z remote: Counting objects: 100% (23/23)        
2022-09-01T18:21:48.7538990Z remote: Counting objects: 100% (23/23), done.        
2022-09-01T18:21:48.7539558Z remote: Compressing objects:   6% (1/15)        
2022-09-01T18:21:48.7540167Z remote: Compressing objects:  13% (2/15)        
2022-09-01T18:21:48.7540664Z remote: Compressing objects:  20% (3/15)        
2022-09-01T18:21:48.7541705Z remote: Compressing objects:  26% (4/15)        
2022-09-01T18:21:48.7542313Z remote: Compressing objects:  33% (5/15)        
2022-09-01T18:21:48.7543334Z remote: Compressing objects:  40% (6/15)        
2022-09-01T18:21:48.7543816Z remote: Compressing objects:  46% (7/15)        
2022-09-01T18:21:48.7544621Z remote: Compressing objects:  53% (8/15)        
2022-09-01T18:21:48.7545096Z remote: Compressing objects:  60% (9/15)        
2022-09-01T18:21:48.7545879Z remote: Compressing objects:  66% (10/15)        
2022-09-01T18:21:48.7546341Z remote: Compressing objects:  73% (11/15)        
2022-09-01T18:21:48.7547151Z remote: Compressing objects:  80% (12/15)        
2022-09-01T18:21:48.7547624Z remote: Compressing objects:  86% (13/15)        
2022-09-01T18:21:48.7548482Z remote: Compressing objects:  93% (14/15)        
2022-09-01T18:21:48.7548933Z remote: Compressing objects: 100% (15/15)        
2022-09-01T18:21:48.7549706Z remote: Compressing objects: 100% (15/15), done.        
2022-09-01T18:21:48.7550476Z remote: Total 23 (delta 2), reused 18 (delta 1), pack-reused 0        
2022-09-01T18:21:48.7624747Z From https://github.com/UriKatsirPrivate/Feature-Environment
2022-09-01T18:21:48.7625577Z  * [new ref]         1e34e46b5a72649fb69427f76647278dff8e5e99 -> origin/main
2022-09-01T18:21:48.7649303Z ##[endgroup]
2022-09-01T18:21:48.7650056Z ##[group]Determining the checkout info
2022-09-01T18:21:48.7651391Z ##[endgroup]
2022-09-01T18:21:48.7652009Z ##[group]Checking out the ref
2022-09-01T18:21:48.7656485Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2022-09-01T18:21:48.7707578Z Switched to a new branch 'main'
2022-09-01T18:21:48.7708322Z branch 'main' set up to track 'origin/main'.
2022-09-01T18:21:48.7713614Z ##[endgroup]
2022-09-01T18:21:48.7753238Z [command]/usr/bin/git log -1 --format='%H'
2022-09-01T18:21:48.7781742Z '1e34e46b5a72649fb69427f76647278dff8e5e99'
2022-09-01T18:21:48.8081639Z ##[group]Run google-github-actions/auth@v0
2022-09-01T18:21:48.8081918Z with:
2022-09-01T18:21:48.8082268Z   workload_identity_provider: projects/854735162550/locations/global/workloadIdentityPools/github-pool/providers/my-provider
2022-09-01T18:21:48.8082720Z   service_account: workflows-sa@landing-zone-demo-341118.iam.gserviceaccount.com
2022-09-01T18:21:48.8083050Z   create_credentials_file: true
2022-09-01T18:21:48.8083296Z   export_environment_variables: true
2022-09-01T18:21:48.8083527Z   cleanup_credentials: true
2022-09-01T18:21:48.8083758Z   access_token_lifetime: 3600s
2022-09-01T18:21:48.8084080Z   access_token_scopes: https://www.googleapis.com/auth/cloud-platform
2022-09-01T18:21:48.8084360Z   retries: 0
2022-09-01T18:21:48.8084561Z   id_token_include_email: false
2022-09-01T18:21:48.8084760Z env:
2022-09-01T18:21:48.8085069Z   PROJECT_ID: landing-zone-demo-341118
2022-09-01T18:21:48.8085293Z   SERVICE: floor
2022-09-01T18:21:48.8085499Z   REGION: europe-west4
2022-09-01T18:21:48.8085731Z ##[endgroup]
2022-09-01T18:21:49.0519917Z Created credentials file at "/home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json"
2022-09-01T18:21:49.0520265Z 
2022-09-01T18:21:49.0543455Z 
2022-09-01T18:21:49.0618550Z ##[group]Run google-github-actions/setup-gcloud@v0
2022-09-01T18:21:49.0618827Z with:
2022-09-01T18:21:49.0619015Z   version: latest
2022-09-01T18:21:49.0619232Z   export_default_credentials: false
2022-09-01T18:21:49.0619476Z   cleanup_credentials: true
2022-09-01T18:21:49.0619686Z env:
2022-09-01T18:21:49.0619900Z   PROJECT_ID: landing-zone-demo-341118
2022-09-01T18:21:49.0620142Z   SERVICE: floor
2022-09-01T18:21:49.0620349Z   REGION: europe-west4
2022-09-01T18:21:49.0620742Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json
2022-09-01T18:21:49.0621270Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json
2022-09-01T18:21:49.0621755Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json
2022-09-01T18:21:49.0622243Z   CLOUDSDK_CORE_PROJECT: landing-zone-demo-341118
2022-09-01T18:21:49.0622574Z   CLOUDSDK_PROJECT: landing-zone-demo-341118
2022-09-01T18:21:49.0623017Z   GCLOUD_PROJECT: landing-zone-demo-341118
2022-09-01T18:21:49.0623289Z   GCP_PROJECT: landing-zone-demo-341118
2022-09-01T18:21:49.0623576Z   GOOGLE_CLOUD_PROJECT: landing-zone-demo-341118
2022-09-01T18:21:49.0623818Z ##[endgroup]
2022-09-01T18:21:49.8239411Z [command]/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/11ed0d2e-2e93-4675-9e87-4f1540bcb665 -f /home/runner/work/_temp/57151bfe-cffe-4f8a-bb8e-8731c6cb12c8
2022-09-01T18:22:03.5171490Z ##[group]Run gcloud builds submit /floor/ --tag europe-west4-docker.pkg.dev/landing-zone-demo-341118/cloud-run-source-deploy/feature-environment:latest
2022-09-01T18:22:03.5172237Z [36;1mgcloud builds submit /floor/ --tag europe-west4-docker.pkg.dev/landing-zone-demo-341118/cloud-run-source-deploy/feature-environment:latest[0m
2022-09-01T18:22:03.5234976Z shell: /usr/bin/bash -e {0}
2022-09-01T18:22:03.5235194Z env:
2022-09-01T18:22:03.5235425Z   PROJECT_ID: landing-zone-demo-341118
2022-09-01T18:22:03.5235670Z   SERVICE: floor
2022-09-01T18:22:03.5235874Z   REGION: europe-west4
2022-09-01T18:22:03.5236264Z   CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json
2022-09-01T18:22:03.5236788Z   GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json
2022-09-01T18:22:03.5237314Z   GOOGLE_GHA_CREDS_PATH: /home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json
2022-09-01T18:22:03.5237698Z   CLOUDSDK_CORE_PROJECT: landing-zone-demo-341118
2022-09-01T18:22:03.5237992Z   CLOUDSDK_PROJECT: landing-zone-demo-341118
2022-09-01T18:22:03.5238262Z   GCLOUD_PROJECT: landing-zone-demo-341118
2022-09-01T18:22:03.5238531Z   GCP_PROJECT: landing-zone-demo-341118
2022-09-01T18:22:03.5238802Z   GOOGLE_CLOUD_PROJECT: landing-zone-demo-341118
2022-09-01T18:22:03.5239119Z   CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
2022-09-01T18:22:03.5239389Z ##[endgroup]
2022-09-01T18:22:04.2748100Z ERROR: (gcloud.builds.submit) There was a problem refreshing your current auth tokens: ('Unable to acquire impersonated credentials: No access token or invalid expiration in response.', '{\n  "error": {\n    "code": 403,\n    "message": "The caller does not have permission",\n    "status": "PERMISSION_DENIED"\n  }\n}\n')
2022-09-01T18:22:04.2748717Z Please run:
2022-09-01T18:22:04.2748848Z 
2022-09-01T18:22:04.2749051Z   $ gcloud auth login
2022-09-01T18:22:04.2749187Z 
2022-09-01T18:22:04.2749281Z to obtain new credentials.
2022-09-01T18:22:04.2749420Z 
2022-09-01T18:22:04.2749559Z If you have already logged in with a different account:
2022-09-01T18:22:04.2749730Z 
2022-09-01T18:22:04.2749827Z     $ gcloud config set account ACCOUNT
2022-09-01T18:22:04.2749980Z 
2022-09-01T18:22:04.2750110Z to select an already authenticated account to use.
2022-09-01T18:22:04.3514119Z ##[error]Process completed with exit code 1.
2022-09-01T18:22:04.3560273Z Post job cleanup.
2022-09-01T18:22:04.4016549Z Skipping credential cleanup - "export_default_credentials" is false.
2022-09-01T18:22:04.4120197Z Post job cleanup.
2022-09-01T18:22:04.4831879Z Removed exported credentials at "/home/runner/work/Feature-Environment/Feature-Environment/gha-creds-1abbc6b1f9e9165c.json".
2022-09-01T18:22:04.4900636Z Post job cleanup.
2022-09-01T18:22:04.6184912Z [command]/usr/bin/git version
2022-09-01T18:22:04.6232742Z git version 2.37.2
2022-09-01T18:22:04.6299611Z Temporarily overriding HOME='/home/runner/work/_temp/452562b5-229b-4185-8d6b-54023bd09f8d' before making global git config changes
2022-09-01T18:22:04.6300304Z Adding repository directory to the temporary git global config as a safe directory
2022-09-01T18:22:04.6301621Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/Feature-Environment/Feature-Environment
2022-09-01T18:22:04.6356596Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2022-09-01T18:22:04.6392927Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2022-09-01T18:22:04.6639498Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2022-09-01T18:22:04.6665616Z http.https://github.com/.extraheader
2022-09-01T18:22:04.6677590Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2022-09-01T18:22:04.6747243Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2022-09-01T18:22:04.7276765Z Evaluate and set job outputs
2022-09-01T18:22:04.7292154Z Cleaning up orphan processes
```

### Additional information

Please note that the "Set up Cloud SDK" step succeeds. Only the step that runs a gcloud command is failing.

Thank you,

Uri

PS: I can be reached internally using my ldap: ukatsir@

**Comment by UriKatsirPrivate** [comment]
One more piece of information: The service account used for auth has permissions to execute the command.

**Comment by UriKatsirPrivate** [comment]
I found my issue.
step #8 in the [Setting up Workload Identity Federation](https://github.com/google-github-actions/auth#setting-up-workload-identity-federation) setup must be repeated for each repo on github.
